package-parser/package_parser/commands/get_api/_refined_types.py

```diff
@@ -134,4 +134,61 @@ def __eq__(self, __o: object) -> bool:
 
 @dataclass
 class UnionType:
-    types: set[Union[NamedType, EnumType, BoundaryType]] = field(default_factory=set)
+    types: list[str]
+
+    @classmethod
+    def from_string(cls, type_str: str) -> UnionType:
+        # Remove all non-necessary whitespaces
+        type_str = re.sub("[ ]+", " ", type_str)
+        # Find all Enums and remove them from doc_string
+        enum_array_matches = re.findall("\{.*?\}", type_str)
+        type_str = re.sub("\{.*?\}", " ", type_str)
+
+        # Remove default-Value from doc_string
+        type_str = re.sub("default=.*", " ", type_str)
+
+        # Create a list with all values and types
+        # ") or (" must be replaced by a very unlikely string ("&%&") so that it is not removed when filtering out.
+        # The string will be replaced by ") or (" again after filtering out.
+
+        type_str = re.sub("\) or \(", "&%&", type_str)
+        type_str = re.sub(" or ", ", ", type_str)
+        type_str = re.sub(" ,[ ]*or ", ", ", type_str)
```

Comment:
Swap these two lines, otherwise we might get `,,`. This does not have an impact later since empty matches are removed but it would be cleaner.